lib/js/components/Icons/FeatureIcon/FeatureIcon.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+	<div
+		class="featureIcon"
+		:class="{
+			'-doubleBackground': doubleBackground,
+			'-danger': color === FEATURE_ICON_COLOR.DANGER,
+			'-info': color === FEATURE_ICON_COLOR.INFO,
+			'-neutral': color === FEATURE_ICON_COLOR.NEUTRAL,
+			'-neutral-weak': color === FEATURE_ICON_COLOR.NEUTRAL_WEAK,
+			'-primary': color === FEATURE_ICON_COLOR.PRIMARY,
+			'-success': color === FEATURE_ICON_COLOR.SUCCESS,
+			'-warning': color === FEATURE_ICON_COLOR.WARNING,
+			'-small': size === FEATURE_ICON_SIZES.SMALL,
+			'-medium': size === FEATURE_ICON_SIZES.MEDIUM,
+			'-large': size === FEATURE_ICON_SIZES.LARGE,
+			'-x-large': size === FEATURE_ICON_SIZES.X_LARGE,
+		}"
+	>
+		<wnl-icon class="featureIcon__icon" :icon="icon" :size="iconSize" />
+	</div>
+</template>
+
+<style lang="scss">
+@import '../../../../styles/components/icons';
+@import '../../../../styles/settings/spacings';
+@import '../../../../styles/settings/colors/tokens';
+
+$feature-icon-colors: (
+	'neutral': (
+		'background': $color-neutral-background-medium,
+		'border': $color-neutral-background,
+		'icon': $color-neutral-icon,
+	),
+	'neutral-weak': (
+		'background': $color-neutral-background-medium,
+		'border': $color-neutral-background,
+		'icon': $color-neutral-icon-weak,
+	),
+	'primary': (
+		'background': $color-primary-background-medium,
+		'border': $color-primary-background,
+		'icon': $color-primary-icon,
+	),
+	'success': (
+		'background': $color-success-background-medium,
+		'border': $color-success-background,
+		'icon': $color-success-icon,
+	),
+	'danger': (
+		'background': $color-danger-background-medium,
+		'border': $color-danger-background,
+		'icon': $color-danger-icon,
+	),
+	'warning': (
+		'background': $color-warning-background-medium,
+		'border': $color-warning-background,
+		'icon': $color-warning-icon,
+	),
+	'info': (
+		'background': $color-info-background-medium,
+		'border': $color-info-background,
+		'icon': $color-info-icon,
+	),
+);
+
+$feature-icon-padding-large: 10px;
```

Comment:
I'm little bit worried that we have to create custom padding for this size. Maybe UX intention was to have fixes size for this component and not use padding at all? Can we confirm what is going on here?

Reply:
Code updated, now we base on the const dimensions, not padding.

Reply:
Agreed with UX, we use custom padding here, as it's more flexible when we change Icon sizes in future